modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -472,7 +472,10 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::checkInputWindow(const GenericReal<is_ad>
     if (behavior == WindowFailure::WARN)
       mooseWarning(msg.str());
     else if (behavior == WindowFailure::ERROR)
-      mooseError(msg.str());
+    {
```

Comment:
You should add an enum `WindowFailure::EXCEPTION`, so the user can specify.

Reply:
I think these should only be exceptions and not errors.  Exceptions will eventually turn into errors after they force the timestep down to dtmin.  The SamplerFullSolveMultiapp is able to catch the errors when they reach dtmin with ignore_solve_not_converge = true and will let that subapp fail without killing the entire mutliapp.  A mooseError will kill the entire multiapp.

Reply:
That is fine, but I still think you need to add the option to throw a hard error. Make another enum for exception, and leave the error as is. Change the default to exception even. It's an easy fix to retain the ability to throw an error

Reply:
A hard error isn't useful, the exception will error out.   All these extra options just get confusing when people copy and paste input files.

Reply:
No, I don't agree, an error may be important to ensure you're passing fair values. This will be important for model development beyond just the roms you are developing.